packages/nextjs/app/builders/0xdaC6E56F86Fb6f18c4114C2E4f41901ab9803096/page.tsx

```diff
@@ -0,0 +1,120 @@
+"use client";
+
+import Image from "next/image";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderPage: NextPage = () => {
+  const size = { width: 32, height: 32 };
+
+  const iconStyle = "w-8 h-8 transition-transform duration-300";
+
+  const anchorStyle = "text-2xl hover:scale-110 transition-transform duration-300 hover:drop-shadow-lg";
+
+  const icons = [
+    {
+      anchorUrl: "https://github.com/NejcRozman",
+      anchorStyle,
+      iconUrl: "https://api.iconify.design/tabler:brand-github.svg",
+      iconAlt: "Github",
+      iconStyle,
+    },
+    {
+      anchorUrl: "https://t.me/NRosca",
+      anchorStyle,
+      iconUrl: "https://api.iconify.design/tabler:brand-telegram.svg",
+      iconAlt: "Telegram",
+      iconStyle,
+    },
+    {
+      anchorUrl: "https://www.linkedin.com/in/nejc-ro%C5%BEman-51567917b/",
+      anchorStyle,
+      iconUrl: "https://api.iconify.design/tabler:brand-linkedin.svg",
+      iconAlt: "LinkedIn",
+      iconStyle,
+    },
+  ];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.